# Request part
float64[] x
float64[] y
bool first
bool last

--- 

# Response part
float64[] constraint_a1
float64[] constraint_a2
float64[] constraint_b
